app/__init__.py

```diff
@@ -4,25 +4,49 @@
 from flask_httpauth import HTTPTokenAuth
 
 from app import version  # noqa
+from app.transformation import Logo
 
 
-LOGO_FILENAMES = {
-    '001': 'hm-government.png',
-    '002': 'opg.png',
-    '003': 'dwp.png',
-    '004': 'geo.png',
-    '005': 'ch.png',
-    '006': 'dwp-welsh.png',
-    '007': 'dept-for-communities.png',
-    '008': 'mmo.png',
-    '500': 'hm-land-registry.png',
+LOGOS = {
```

Comment:
Was there a reason to keep the 001, 002 scheme? It would make more sense to label them as HM_GOV, OPG etc

Reply:
We need to stay compatible with what’s stored in the database for now. Agree that we should revisit this once we’re not dependent on DVLA storing copies of the logos (which they identify with the 001, 002 scheme.